lib/prompts/list.js

```diff
@@ -38,7 +38,8 @@ function Prompt() {
   if (_.isNumber(def) && def >= 0 && def < this.opt.choices.realLength) {
     this.selected = def;
   } else if (!_.isNumber(def) && def != null) {
-    this.selected = this.opt.choices.pluck('value').indexOf(def);
+    let index = this.opt.choices.pluck('value').indexOf(def);
```

Comment:
Thanks for sending the fix. This code would be cleaner as:

```js
this.selected = Math.max(this.opt.choices.pluck('value').indexOf(def), 0);
```

Mind updating?

Reply:
Great catch, I'll use that from now on. Will update it in a while.

Reply:
Updated.